src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/src/Client/ElectricityMapsClient.cs

```diff
@@ -189,6 +190,20 @@ private async Task<HttpResponseMessage> GetResponseAsync(string uriPath)
         if (!response.IsSuccessStatusCode)
         {
             var content = await response.Content.ReadAsStringAsync();
+
+            // Add extra error messaging for free trial users to give more context
```

Comment:
Wouldnt this information be useful for paid subscribers too? For eg, with the current EM subscription, we only have access to specific zones and this clear error message would be helpful. Thoughts?

Reply:
Good point, I've updated to trigger for all users